xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1387,6 +1387,110 @@ class FlashAttentionPagedAttention : public MultiHeadedAttentionTest {
+  const std::string
+  GetModuleFlash_Attention_Flex_Attention_Soft_Capping_BF16() {
+    const std::string hlo_text = R"(
+    HloModule jit__unnamed_wrapped_function_
+
+    %soft_cap.14 (Arg_0.15: f32[4,4,1024,1024]) -> f32[4,4,1024,1024] {
+      %Arg_0.15 = f32[4,4,1024,1024]{3,2,1,0} parameter(0)
+      %constant.1 = f32[] constant(0.333333343)
+      %broadcast.1 = f32[4,4,1024,1024]{3,2,1,0} broadcast(%constant.1), dimensions={}
+      %multiply.1 = f32[4,4,1024,1024]{3,2,1,0} multiply(%Arg_0.15, %broadcast.1)
+      %tanh.19 = f32[4,4,1024,1024]{3,2,1,0} tanh(%multiply.1)
+      %constant.16 = f32[] constant(3)
+      %broadcast.17 = f32[4,4,1024,1024]{3,2,1,0} broadcast(%constant.16), dimensions={}
+      ROOT %multiply.20 = f32[4,4,1024,1024]{3,2,1,0} multiply(%tanh.19, %broadcast.17)
+    }
+
+    ENTRY %main.25 (Arg_0.1: bf16[4,1024,4,64], Arg_1.2: bf16[4,1024,4,64], Arg_2.3: bf16[4,1024,4,64]) -> bf16[4,1024,4,64] {
+      %Arg_0.1 = bf16[4,1024,4,64]{3,2,1,0} parameter(0)
+      %Arg_1.2 = bf16[4,1024,4,64]{3,2,1,0} parameter(1)
+      %Arg_2.3 = bf16[4,1024,4,64]{3,2,1,0} parameter(2)
+      %custom-call.21 = (bf16[4,4,1024,64]{3,1,2,0}, u8[0]{0}) custom-call(%Arg_0.1, %Arg_1.2, %Arg_2.3), custom_call_target="__cudnn$fmhaSoftmax", api_version=API_VERSION_STATUS_RETURNING, called_computations={%soft_cap.14}, backend_config={"operation_queue_id": "0", "wait_on_operation_queues": [], "cudnn_fmha_backend_config": {"algorithm": {"algo_id": "0", "math_type": "TENSOR_OP_MATH", "tuning_knobs": {"17": "1", "24": "0"}, "is_cudnn_frontend": true, "workspace_size": "0"}, "fmha_scale": 1.0, "intermediate_tensor_shape": {"element_type": "BF16", "dimensions": ["4", "4", "1024", "1024"], "tuple_shapes": [], "layout": {"dim_level_types": [], "dim_unique": [], "dim_ordered": [], "minor_to_major": ["3", "2", "1", "0"], "tiles": [], "element_size_in_bits": "0", "memory_space": "0", "index_primitive_type": "PRIMITIVE_TYPE_INVALID", "pointer_primitive_type": "PRIMITIVE_TYPE_INVALID", "dynamic_shape_metadata_prefix_bytes": "0"}, "is_dynamic_dimension": [false, false, false, false]}, "is_flash_attention": true, "mask_type": "NO_MASK", "bmm1_dot_dimension_numbers": {"lhs_contracting_dimensions": ["3"], "rhs_contracting_dimensions": ["3"], "lhs_batch_dimensions": ["0", "2"], "rhs_batch_dimensions": ["0", "2"]}, "bmm2_dot_dimension_numbers": {"lhs_contracting_dimensions": ["3"], "rhs_contracting_dimensions": ["1"], "lhs_batch_dimensions": ["0", "1"], "rhs_batch_dimensions": ["0", "2"]}, "dropout_rate": 0, "seed": 42, "sliding_window_length": 0, "max_seg_per_batch": 1}}
+      %get-tuple-element.22 = bf16[4,4,1024,64]{3,1,2,0} get-tuple-element(%custom-call.21), index=0
+      ROOT %transpose.24 = bf16[4,1024,4,64]{3,2,1,0} transpose(%get-tuple-element.22), dimensions={0,2,1,3}
+    }
+  )";
+    return hlo_text;
+  }
+
+  const std::string
+  GetModuleFlash_Attention_Flex_Attention_Soft_Capping_Reference_BF16() {
+    const std::string hlo_text = R"(
```

Comment:
Can you make this variable a `static constexpr absl::string_view kHloText = ...` and change the return type to `absl::string_view`?